service/router.go

```diff
@@ -184,25 +184,10 @@ func setupRoutes(router *mux.Router, oas *openapi.OpenAPISpec, env config.Enviro
 		}
 	}
 
+	paths, methodsMap, ignoreTrailingSlashMap := openapi.CreateOASUtilityMaps(oas)
```

Comment:
ok the function extraction but we should keep this function here in the service package to prevent coupling.

If it's necessary to keep the function in the openapi package let's define it as a method for the openApi typ

Reply:
rename to `unwrapConfig` and set as a method of oas